### DISCUSSION TITLE
After merging a PR I'd like to easily cherry-pick to other branches

### DISCUSSION THREAD

**Discussion by madrob** [community/community#28206] [Pull Requests]
Right now if I want to create additional PRs for the same fix, I have to make the cherry-pick locally, push new branches, and open new PRs. It would be nice if there was a single cherry pick button after I merge (especially squash merge) and I could select the branch I want to target from a drop down.

**Comment by OneCyrus** [comment]
I would really like to see this feature on GitHub. We use it a lot on Azure DevOps. It reduces the effort quite a lot to backport certain fixes to different release branches.
